velox/docs/functions/spark/datetime.rst

```diff
@@ -22,6 +22,12 @@ These functions support TIMESTAMP and DATE input types.
     ``day`` need to be from 1 to 31, and matches the number of days in each month.
     days of ``year-month-day - 1970-01-01`` need to be in the range of INTEGER type.
 
+.. function:: week(x) -> integer
+
+    Returns the `ISO-Week`_ of the year from x. The value ranges from ``1`` to ``53``.
```

Comment:
In Spark, `A week is considered to start on a Monday and week 1 is the first week with >3 days.`
Can we document this also? And is current implementation the same with above statement?

Reply:
`A week is considered to start on a Monday and week 1 is the first week with >3 days.` this is described in `ISO-Week`.
It's covered by the ut. `1919-12-31` => `1`; `1920-01-01` => `1`

Reply:
Got it. Can we record this in the doc?

> A week is considered to start on a Monday and week 1 is the first week with >3 days.